net/fetcher.go

```diff
@@ -22,6 +30,212 @@ type Fetcher interface {
 	FetchTipSets(ctx context.Context, tsKey types.TipSetKey, from peer.ID, done func(ts types.TipSet) (bool, error)) ([]types.TipSet, error)
 }
 
+// GraphSyncFetcher is used to fetch data over the network.  It is implemented with
+// a persistent bitswap session on a networked blockservice.
+type GraphSyncFetcher struct {
+	gs        *graphsync.GraphSync
+	validator consensus.BlockSyntaxValidator
+	bridge    ipldbridge.IPLDBridge
+	bs        bstore.Blockstore
```

Comment:
`bs` -> `store`?

Reply:
what is the general guideline for you all on variable names? I tend to lean towards the verbose, but I know go lang folks tend to like things short. I noticed some short names in the closest code and started heading that way, but I think maybe from your comment longer is ok?

Reply:
A wide range is acceptable and this is by no means a blocking comment. I think my preference is slightly more descriptive than some.

In general I consider it a function of scope. For a local in a loop or short function, small is great. Struct members need to make sense perhaps 100s of lines away from their definition, so more information helps. Exported members even more so, such that I tend to use one or more actual words for them.